CHANGES.rst

```diff
@@ -97,6 +95,12 @@ skymatch
 
 - Updated to populate the "BKGMETH" keyword in output files. [#6736]
 
+source_catalog
+--------------
+
+- Fixed the KDTree calculation to use only non-finite source positions
```

Comment:
I think you mean *finite* here! We're NOT using the non-finite positions.

Reply:
Thanks! 😃